sources/install.sh

```diff
@@ -239,6 +239,18 @@ function install_php_filter_chain_generator() {
   add-test-command "php_filter_chain_generator --help"
 }
 
+function install_badsecrets() {
+  colorecho "Installing badsecrets"
+  git -C /opt/tools/ clone https://github.com/blacklanternsecurity/badsecrets.git
+  cd /opt/tools/badsecrets
+  python3 -m venv venv && . venv/bin/activate      # Using a venv to avoid conflicts with current versions of pycrypto installed 
+  pip install --no-cache-dir -r requirements.txt
+  pip install badsecrets
```

Comment:
I could, but from my understanding it would change nothing as we are already in an activated python venv (we ran bin/activate just before), please correct me if I'm wrong

Reply:
I don't if it changes anything, my request for change was initially made for uniformity reasons. All tool install functions use `python3 -m pip` instead of `pip` or `pip3`, and I find it more explicit, hence preferred, that's why

Reply:
Thanks for the explanation, I will commit the changes asap